docs/configuration.rst

```diff
@@ -14,11 +14,11 @@ To use a custom configuration file, use the ``-c`` option:
 If ``-c`` is not provided, yamllint will look for a configuration file in the
 following locations (by order of preference):
 
-- ``.yamllint``, ``.yamllint.yaml`` or ``.yamllint.yml`` in the current working
-  directory
-- the file referenced by ``$YAMLLINT_CONFIG_FILE``, if set
-- ``$XDG_CONFIG_HOME/yamllint/config``
-- ``~/.config/yamllint/config``
+- A file named ``.yamllint``, ``.yamllint.yaml``, or ``.yamllint.yml`` in the
```

Comment:
Can you start each bullet item with a lower-case letter? Because the grammatical sentence starts on line 14 and ends on line 21 (by the way, there could be a period there).

-- one explanation amongst others: https://english.stackexchange.com/a/15132